db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek.rb

```diff
@@ -395,16 +403,18 @@
                        id: :v30,
                        hidden: true,
                        type: :range,
-                       title: 'Hoe groot is het gevolg van deze maatregel (2) voor uw dagelijkse werkzaamheden?',
-                       labels: ['Heel klein', 'Heel groot']
+                       title: 'Hoe groot is het gevolg van deze maatregel voor uw dagelijkse werkzaamheden?',
+                       labels: ['Heel klein', 'Heel groot'],
+                       tooltip: maatregel2
                      }, {
                        id: :v31,
                        hidden: true,
                        type: :range,
-                       title: 'Wat vindt u van de gevolgen van deze maatregel (2) voor uw dagelijkse werkzaamheden?',
-                       labels: ['Heel negatief', 'Heel positief']
+                       title: 'Wat vindt u van de gevolgen van deze maatregel voor uw dagelijkse werkzaamheden?',
+                       labels: ['Heel negatief', 'Heel positief'],
+                       tooltip: maatregel2
                      }, {
-                       section_start: '<div class="divider"></div><p class="flow-text"><strong>Maatregel 3</strong>: De RMC contactgemeenten hebben de verantwoordelijkheid gekregen voor de totstandkoming van de regionale samenwerking.</p>',
+                       section_start: '<div class="divider"></div><p class="flow-text"><em>De volgende vragen gaan over onderstaande maatregel.</em></p>' + maatregel3,
```

Comment:
deze stukken kun je ook nog in een variabele zetten
`maatregel_text` + `maatregel1`